p2p/src/authenticated/discovery/actors/peer/actor.rs

```diff
@@ -204,12 +204,16 @@ impl<E: Spawner + Clock + ReasonablyRealtime + Rng + CryptoRng + Metrics, C: Pub
                         types::Payload::BitVec(_) => &bit_vec_rate_limiter,
                         types::Payload::Peers(_) => &peers_rate_limiter,
                         types::Payload::Data(data) => {
-                            let Some(rate_limiter) = rate_limits.get(&data.channel) else {
-                                // We permit unknown messages to be received in case peers
-                                // are on a newer version than us
-                                continue;
-                            };
-                            rate_limiter
+                            match rate_limits.get(&data.channel) {
+                                Some(rate_limit) => rate_limit,
+                                None => { // Treat unknown channels as malformed
+                                    info!(?peer, channel = data.channel, "unknown channel");
```

Comment:
We should make this a `debug!` log. It would be trivial for a peer to force us to emit logs.

Reply:
The `info` above is a bug.

Reply:
We emit a debug when connecting to a peer and I feel they should at least by symmetric